### DISCUSSION TITLE
How to use a 4G module to connect to the network and transmit audio recordings?

### DISCUSSION THREAD

**Discussion by Merle-h** [mcguirepr89/BirdNET-Pi#1247] [Q&A]
Hi,

I’ve assembled a BirdNET Pi using a Raspberry Pi 4B, a BOYA MM1 microphone, and a UGREEN USB to 3.5mm external sound card. Initially, I ran it on my home balcony and connected it to the network using an Ethernet cable to my Wi-Fi router. Now, I’m planning to install this device in my school’s biology garden. I can change the power bank daily for power supply, but a wired network connection is difficult to achieve.

I’ve consulted with some experienced folks, and they suggested using a 4G module to establish a network connection. However, I haven’t been able to find a tutorial or guide on how to achieve this (maybe I’m not searching correctly). So, I’m reaching out here to ask if anyone knows how to implement this method?

 If you could provide any help, I’d greatly appreciate it!

**Comment by CountryTec** [comment]
Am using SIM7670G HAT on the PI4. It was kind of tricky to get it working despite following the respective installation guide.
